### PR TITLE
feat: Update context menu entry if no profiles

### PR DESCRIPTION
**Pull request by DanSM-5**
Update to change to a single entry in context menu if no profiles.

Solves: https://github.com/woodruffw/ff2mpv/issues/114

### CODE REVIEW

**Comment by woodruffw** [comment]
Thanks! Will review tonight.

File: ff2mpv.js [hunk 1]
```diff
@@ -32,68 +32,106 @@ async function getOptions(id) {
...
+    const url = info.linkUrl || info.srcUrl || info.selectionText || info.frameUrl;
+    if (url) {
+      const options = await getOptions(info.menuItemId);
+      ff2mpv(url, options);
+    }
   }
 }
 
-function createProfile(profile) {
+let os = '';
```
Comment (woodruffw) [inline ff2mpv.js:47]: This mutable global state feels a little funny -- can't we call `getOS` twice or, even better, call it just once to set a global `TITLE` constant based on the right host?

**Review by woodruffw — COMMENTED** [review]
One comment/question, otherwise this looks pretty good to me. Thanks @DanSM-5!

File: ff2mpv.js (hunk 1 above)
Comment (DanSM-5) [reply ff2mpv.js]: I've defined `TITLE` as a global variable. Let me know if it looks better know.

**Review by woodruffw — APPROVED** [review]

**Pull request merged**